package.json

```diff
@@ -83,6 +83,7 @@
     "eslint-plugin-mdx": "^1.16.0",
     "glob": "^7.1.4",
     "graphql": "^15.8.0",
+    "graphql-ws": "^5.12.0",
```

Comment:
what is this for?

Reply:
peer dep
![image](https://user-images.githubusercontent.com/15986172/223518429-6ef88ae9-268c-4985-90de-935fd201340a.png)